diff --git a/node_modules/feed-reader/src/main.js b/node_modules/feed-reader/src/main.js
index 595a0e8..514fc74 100755
--- a/node_modules/feed-reader/src/main.js
+++ b/node_modules/feed-reader/src/main.js
@@ -5,8 +5,8 @@
 
 import { info } from './utils/logger.js'
 
-import getXML from './utils/retrieve.js'
-import xml2obj from './utils/xml2obj.js'
+// import getXML from './utils/retrieve.js' // issue here
+// import xml2obj from './utils/xml2obj.js' // and here
 import { parseRSS, parseAtom } from './utils/parser.js'
 
 import {
@@ -21,15 +21,15 @@ export {
 } from './config.js'
 
 export const read = async (url) => {
-  const xmldata = await getXML(url)
-  if (!xmldata) {
-    throw new Error(`Could not fetch XML content from "${url}"`)
-  }
-  const { xml } = xmldata
-  if (!validate(xml)) {
-    throw new Error(`Failed while validating XML format from "${url}"`)
-  }
-  info('Parsing XML data...')
-  const jsonObj = xml2obj(xml)
-  return isRSS(jsonObj) ? parseRSS(jsonObj) : isAtom(jsonObj) ? parseAtom(jsonObj) : null
+  // const xmldata = await getXML(url)
+  // if (!xmldata) {
+  //   throw new Error(`Could not fetch XML content from "${url}"`)
+  // }
+  // const { xml } = xmldata
+  // if (!validate(xml)) {
+  //   throw new Error(`Failed while validating XML format from "${url}"`)
+  // }
+  // info('Parsing XML data...')
+  // const jsonObj = xml2obj(xml)
+  // return isRSS(jsonObj) ? parseRSS(jsonObj) : isAtom(jsonObj) ? parseAtom(jsonObj) : null
 }
